provision/examples/azure/README.md

```diff
@@ -10,12 +10,14 @@ This example shows you how to use Hydroform to provision an Azure cluster.
 
 To provision an Azure cluster you need:
 
-1. The **Subscription ID**, **Tenant ID**, **Client ID** and **Client Secret** of your Azure subscription stored in a TOML file:
-    ```toml
-    SUBSCRIPTION_ID = "<my-subscription-id>"
-    TENANT_ID = "<my-tenant-id>"
-    CLIENT_ID = "<my-client-id>"
-    CLIENT_SECRET = "<my-client-secret>"
+1. The **Subscription ID**, **Tenant ID**, **Client ID** and **Client Secret** of your Azure subscription stored in a JSON file:
```

Comment:
```suggestion
1. The **Subscription ID**, **Tenant ID**, **Client ID**, and **Client Secret** of your Azure subscription stored in a JSON file:
```

Reply:
serial comma? ;)

Reply:
I would not bold them if unless you specifically quote the exact parameter names (and from what I can see, you don't)

Reply:
Fixed thanks!